src/main/java/dlt/dltbackendmaster/controller/BeneficiaryController.java

```diff
@@ -35,6 +35,15 @@ public ResponseEntity<List<Beneficiaries>> getAll() {
 
         try {
             List<Beneficiaries> beneficiaries = service.getAll(Beneficiaries.class);
+
+            for (Beneficiaries beneficiary : beneficiaries) {
```

Comment:
please evaluate the proposal of leaving partner Id, avoiding the for clause

Reply:
Loop excluded as per recomendation